src/applications/financial-status-report/wizard/pages/Disagree.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { PAGE_NAMES } from '../constants';
+
+const Disagree = () => {
+  return (
+    <div className="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-top--2">
+      <p className="vads-u-margin-top--0">
+        Based on the information you provided, this isn’t the form you need.
+      </p>
+      <p>
+        <strong>
+          If you disagree with the VA decision that resulted in this debt,
+        </strong>
+        you can submit a supplemental claim or request a higher level review or
+        board appeal.
+        <a className="wizard-links" href="/decision-reviews/">
+          Learn more about the VA appeals process
+        </a>
+      </p>
+      <p>
+        <strong>If you need more help, </strong>
+        call your VA benefit office.
+        <a className="wizard-links" href="/resources/helpful-va-phone-numbers/">
```

Comment:
Is this `wizard-links` class for all links? I didn't see it in any of the other links on other wizard pages.

Reply:
it was meant to be a class to style some links throughout the wizard but may possibly not be needed ill look into it tks